### PR TITLE
[RHCLOUD-23884] Update RBAC email templates

### PR DESCRIPTION
**Pull request by g-duval**
image

image

image

image

image

image

image

image

image

image

image

image

image

### CODE REVIEW

File: engine/src/main/resources/templates/Rbac/platformGroupToCustomEmailBodyV2.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+{#include Common/insightsEmailBody}
+{#content-title}
+    User Access - Console
+{/content-title}
+{#content-title-section1}
+    User Access platform group Notification
+{/content-title-section1}
+{#content-body-section1}
+    <p>
+        Platform default group is modified by {action.events[0].payload.username} and the Red Hat will not be responsible for managing it from now on.
```
Comment (gwenneg) [inline engine/src/main/resources/templates/Rbac/platformGroupToCustomEmailBodyV2.html:12]: There's a typo in the old sentence:
```suggestion
        Platform default group is modified by {action.events[0].payload.username} and Red Hat will not be responsible for managing it from now on.
```

File: engine/src/main/resources/templates/Rbac/platformGroupToCustomEmailBodyV2.html [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{@boolean renderSection1=true}
+{@boolean renderButtonSection1=true}
+{#include Common/insightsEmailBody}
+{#content-title}
+    User Access - Console
+{/content-title}
+{#content-title-section1}
+    User Access platform group Notification
```
Comment (gwenneg) [inline engine/src/main/resources/templates/Rbac/platformGroupToCustomEmailBodyV2.html:8]: Is `Notification` necessary? It was removed from other templates.
```suggestion
    User Access platform group
```

Comment (g-duval) [reply engine/src/main/resources/templates/Rbac/platformGroupToCustomEmailBodyV2.html:8]: Seems useless indeed

File: common/src/main/java/com/redhat/cloud/notifications/config/FeatureFlipper.java [hunk 3]
```diff
@@ -137,6 +140,7 @@ void logFeaturesStatusAtStartup(@Observes StartupEvent event) {
         Log.infof("The Resource optimization's email templates V2 are %s", resourceOptimizationManagementEmailTemplatesV2Enabled ? "enabled" : "disabled");
         Log.infof("The Vulnerability's email templates V2 are %s", vulnerabilityEmailTemplatesV2Enabled ? "enabled" : "disabled");
         Log.infof("The Rhosak's email templates V2 are %s", rhosakEmailTemplatesV2Enabled ? "enabled" : "disabled");
+        Log.infof("The Cost management's email templates V2 are %s", rbacEmailTemplatesV2Enabled ? "enabled" : "disabled");
```
Comment (gwenneg) [inline common/src/main/java/com/redhat/cloud/notifications/config/FeatureFlipper.java:143]:
```suggestion
        Log.infof("The RBAC's email templates V2 are %s", rbacEmailTemplatesV2Enabled ? "enabled" : "disabled");
```

**Review by gwenneg — APPROVED** [review]
LGTM

**Pull request merged**